components/Author/Tabs/FeedCard.tsx

```diff
@@ -0,0 +1,558 @@
+import Router from "next/router";
+import colors, { genericCardColors } from "~/config/themes/colors";
+import dynamic from "next/dynamic";
+import icons from "~/config/themes/icons";
+import { DOWNVOTE, UPVOTE, userVoteToConstant } from "~/config/constants";
+import { ModalActions } from "~/redux/modals";
+import { PaperActions } from "~/redux/paper";
+import { SyntheticEvent, useState, useEffect } from "react";
+import { breakpoints } from "~/config/themes/screen";
+import { connect } from "react-redux";
+import { css, StyleSheet } from "aphrodite";
+import { emptyFncWithMsg, isNullOrUndefined } from "~/config/utils/nullchecks";
+import { formatDate } from "~/config/utils/dates";
+import { isDevEnv } from "~/config/utils/env";
+import { transformDate } from "~/redux/utils";
+
+const PaperPDFModal = dynamic(
+  () => import("~/components/Modals/PaperPDFModal")
+);
+
+export type FeedCardProps = {
+  boost_amount: number;
+  created_by: any;
+  created_date: any;
+  discussion_count: number;
+  first_figure: any;
+  first_preview: any;
+  formattedDocType: string | null;
+  hubs: any[];
+  id: number;
+  index: number;
+  onBadgeClick: any;
+  openPaperPDFModal: any;
+  paper: any;
+  postDownvote: any;
+  postUpvote: any;
+  preview_img: string;
+  renderableTextAsHtml: any;
+  renderable_text: string;
+  score: number;
+  singleCard: boolean;
+  slug: string;
+  style: StyleSheet;
+  styleVariation: string;
+  title: string;
+  titleAsHtml: any;
+  unified_document: any;
+  unified_document_id: number;
+  uploaded_by: any;
+  uploaded_date: any;
+  user: any;
+  user_vote: any;
+  voteCallback: any;
+};
+
+function FeedCard(props: FeedCardProps) {
+  const {
+    boost_amount: boostAmount,
+    created_by,
+    created_date,
+    discussion_count,
+    first_figure,
+    first_preview,
+    formattedDocType,
+    hubs,
+    id,
+    index,
+    onBadgeClick,
+    paper,
+    postDownvote,
+    postUpvote,
+    preview_img: previewImg,
+    renderableTextAsHtml,
+    renderable_text: renderableText,
+    score: initialScore,
+    singleCard,
+    slug,
+    style,
+    styleVariation,
+    title,
+    titleAsHtml, // In some contexts we want to wrap the title/renderable_text with html. e.g. rendering search highlights.
+    unified_document: unifiedDocument,
+    unified_document_id: unifiedDocumentId,
+    uploaded_by,
+    uploaded_date,
+    user: currentUser,
+    user_vote: userVote,
+    voteCallback,
+  } = props;
+
+  /**
+   * Whether or not THIS PaperPDFModal is open.
+   * There may be many PaperPDFModal components on the page, but
+   * modals.openPaperPDFModal is only a single boolean. So all cards
+   * must only render their PaperPDFModal component if requested */
+  const [isPreviewing, setIsPreviewing] = useState(false);
+  const [voteState, setVoteState] = useState<string | null>(
+    userVoteToConstant(userVote)
+  );
+  const [score, setScore] = useState<number>(initialScore + (boostAmount || 0));
```

Comment:
@thomas-vu we are moving away from adding boost amount in the score because it is confusing to the user and also to us when we debug. I made that change on the post page but didn't realize we are also doing this on cards - can you remove the boost from the score?

Generally speaking, we should decouple the score from the twitter score and any other factors such as this.

Reply:
Sure thing, I'll remove this.